doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,188 @@
 [<< Back](../)
 
 # 1. Introduction
+<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
+
+* [1.1 Introduction](#1.1)
+  * [1.1.1 Terminology](#1.1)
+  * [1.1.2 Relation to other communities](#1.1)
+* [1.2 Scope](#1.2)
+* [1.3 Principles and Guidelines](#1.3)
+  * [1.4.1 Overarching Objectives and Goals](#1.4.1)
+* [1.4 Best Practices](#1.4)
+* [1.5 Verification methodologies](#1.5)
+*	[1.6 Assumptions & Dependencies](#1.6)
+*	[1.7 Results Collation & Presentation](#1.7)
+* [1.9 Resources & References](#1.9)
+
+## Executive Summary
```

Comment:
Typically, we have not had an "Executive Summary" in Chapters. Would this content fit better under Introduction?

Reply:
This is the same format as RC1 so I'm fine with the structure unless anyone else has a problem with it